src/lib/config.js

```diff
@@ -2,15 +2,47 @@ import deepClone from 'deep-clone'
 import deepEqual from 'deep-equal'
 
 import stack from '../lib/stack'
-import addComingSoonApps from './comingSoon'
 
 import MENU_CONFIG from '../config/menu'
 
 const EXCLUDES = ['settings', 'onboarding']
 const CATEGORIES = ['cozy', 'partners', 'ptnb']
 
+let cachedComingSoonApps
+function fetchComingSoonApps () {
+  if (cachedComingSoonApps) return Promise.resolve(cachedComingSoonApps)
+  return stack.get.context()
+    .then(context => {
+      const comingSoonApps = context.data && context.data.attributes &&
+        context.data.attributes['coming_soon'] &&
+          Object.values(context.data.attributes['coming_soon']) || []
+
+      cachedComingSoonApps = comingSoonApps.map(app => {
+        let icon
+
+        try {
+          icon = app.slug && {
+            cached: true,
+            src: require(`../assets/icons/comingsoon/icon-${app.slug.toLowerCase()}.svg`)
```

Comment:
I think we shouldn't change the `slug` if we need to use it. If it's not "formatted" correctly or as we want, that means it's not correct at the source and we should change the source and use it as provided. Otherwise it can be confusing as the slug will be formatted differently from the source in the icon name.